dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -243,10 +243,10 @@ BEGIN
     IDS_INVALIDLOGONHOURS "Contul dumneavoastră are restricții de timp care vă împiedică să vă conectați în acest moment. Încercați mai târziu."
     IDS_INVALIDWORKSTATION "Contul dumneavoastră este configurat pentru a vă preveni în a folosi acest calculator. Încercați la alt calculator."
     IDS_ACCOUNTRESTRICTION "Eșec în conectarea dumneavoastră din cauza unei restricții a contului."
-    IDS_EMERGENCYLOGOFFTITLE "Log Off ReactOS"
-    IDS_EMERGENCYLOGOFF "Click OK to immediately log off. Any unsaved data will be lost. Use this only as a last resort."
-    IDS_EMERGENCYRESTARTTITLE "Emergency restart"
-    IDS_EMERGENCYRESTART "Click OK to immediately restart. Any unsaved data will be lost. Use this only as a last resort."
+    IDS_EMERGENCYLOGOFFTITLE "Deautetifică ReactOS"
+    IDS_EMERGENCYLOGOFF "Apăsați OK pentru a vă deautentifica imediat. Toate datele nesalvate vor fi pierdute. Folosiți acest lucru doar ca ultimă soluție."
+    IDS_EMERGENCYRESTARTTITLE "Repornire de urgență"
```

Comment:
Verb mood inconsistency. The mood of this verb should be the same as that found in IDS_EMERGENCYLOGOFFTITLE resource's string, but it isn't (i.e. the two have to either be "Deautentifică" & "Repornește", or "Deautentificare" & "Repornire").

Reply:
I chose "Deautentificare" & "Repornire".